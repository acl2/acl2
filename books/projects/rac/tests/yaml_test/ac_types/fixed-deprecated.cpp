// RAC begin

int foo() {
  ac_fixed<4, 4> a;
  return 0;
}

// RAC end
