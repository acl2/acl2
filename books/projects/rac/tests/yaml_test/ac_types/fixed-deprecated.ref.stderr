fixed-deprecated.cpp:4 (2-10): ac_fixed is deprecated
  4 |   ac_fixed<4, 4> a;
    |   ^^^^^^^^
